lib/Launcher.js

```diff
@@ -97,7 +97,7 @@ class Launcher {
       if (os.platform() === 'win32')
         chromeArguments.push('--disable-gpu');
     }
-    if (!options.ignoreDefaultArgs && Array.isArray(options.args) && options.args.every(arg => arg.startsWith('-')))
```

Comment:
This is a slight functionality change. There is now no way to get the newtab page in headful chrome as the startup tab. That is probably fine though.

Reply:
>  There is now no way to get the newtab page in headful chrome as the startup tab. 

I think you can navigate to `chrome://newtab`